lib/components/TokenDetailsCard.jsx

```diff
@@ -36,15 +37,22 @@ export const TokenDetailsCard = (props) => {
     setSponsorshipName,
     setUserChangedSponsorshipTicker,
     sponsorshipSymbol,
-    setSponsorshipSymbol,
+    setSponsorshipSymbol
   } = props
 
+  const tokenDetailsDescription = classnames(
```

Comment:
This read really confusing to me at first. Let's keep `classnames` for the CSS `className={}` attribute only, and do the same thing you did for RNGCard here.

https://github.com/pooltogether/pooltogether-pool-builder-ui/pull/21/files#diff-ac7e64061261e7a3c17051789407b8f15710a780ca90c28487ee0c7efb16608cR30-R41